bindings/go/scip/symbol.go

```diff
@@ -168,7 +172,12 @@ func (s *symbolParser) parseDescriptor() (*Descriptor, error) {
 		default:
 		}
 	}
-	return nil, nil
+
+	end := s.index
+	if s.index > len(s.Symbol) {
+		end = len(s.Symbol)
+	}
+	return nil, errors.Newf("unrecognized descriptor %q", string(s.Symbol[start:end]))
```

Comment:
Was there a reason this didn't return an error before?

Reply:
probably just overlooked, don't think it was intentional